src/NLPModelsKnitro.jl

```diff
@@ -53,7 +53,7 @@ function _knitro(::Val{true}, nlp :: AbstractNLPModel;
     lvar = nlp.meta.lvar
     if any(lvarinf)
       lvar = copy(nlp.meta.lvar)
-      lvar[lvarinf] .= -KTR_INFINITY
+      lvar[lvarinf] .= -KTR_INFBOUND
```

Comment:
I would recommend using `KN_INFINITY` instead (the previous API whose names start by `KTR` is bound to be deprecated in a few years).

Reply:
Ah, great, thank you!